### PR TITLE
added star-rating

### PR DESCRIPTION
**Pull request by alexanderpalsson**
Added star rating

### CODE REVIEW

**Comment by sublibra** [comment]
Great start! I like 
* the feature :-)
* the comment when hoovering over the stars. 
* the appearance of the textbox when selecting a score.

Some comments:
- I think it is too big overall. It is enough to have it as a small. I would like the size to be more close to https://www.w3schools.com/howto/howto_css_star_rating.asp.
- Is the star rating and comment submit different events? I would like it so but if it is the case also give the user knowledge about this.
- What about changing the text in the textbox to: "Thank you! Please help us improve." and "Thank you! What stands out? Yes / No?

**Comment by hrigner** [comment]
I also would like it to be a bit smaller. And maybe some more top-padding so it gets more distanced to the doc text.

Here it gets a bit misplaced -> https://deploy-preview-532--core-website.netlify.com/why-qlik-core/

**Comment by hrigner** [comment]
It gets a bit crowded with the on-hover-popup. I think we can remove the popup this since star rating is commonly used and it is obvious what it means. Or if we wanna keep it, redesign it a bit.

**Comment by hrigner** [comment]
There are some errors popping up in the console, e.g
`https://deploy-preview-532--core-website.netlify.com/updates/javascripts/submit-textbox.js net::ERR_ABORTED 404`

File: docs/stylesheets/star-rating.css [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+#stars {
+  display: block;
+  width: 125px;
+}
+
+#submitform {
+  border-radius: 4px;
+  display: none;
+  -webkit-animation: fadeIn 1 s;
+  animation: fadeIn 1 s;
```
Comment (hrigner) [inline docs/stylesheets/star-rating.css]: should be `animation: fadeIn 1s;`, without the space in `1 s`. This way the popup will fade in nicely.

File: docs/stylesheets/star-rating.css [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+    opacity: 0;
+  }
+
+  to {
+    opacity: 1;
+  }
+}
+
+/* Add animation (fade in the popup) */
+@-webkit-keyframes fadeIn {
```
Comment (hrigner) [inline docs/stylesheets/star-rating.css]: `@-webkit-keyframes fadeIn` and also `@keyframes fadeIn` seems to be declared twice?

**Comment by hrigner** [comment]
This is nice! :) Just some minor things..

- Maybe add a fadeOut effect when the popup is closed.

- Add some tooltip saying what the stars are there for, shown when hovering over the stars container.

**Pull request merged**